### ISSUE TITLE
Bundle GLFW with Wayland and Libdecor support

### ISSUE THREAD

**Issue by orowith2os** [flathub/org.prismlauncher.PrismLauncher#9]
I'm suggesting this, as it could allow for people to disable X11 entirely in PrismLauncher (on their system, not for the Flathub package) and is generally of interest to Wayland users.

How could this be handled? A Flatpak extension? Bundling it in the main manifest? Or is there some other way to achieve this?

It would also be beneficial to mention to end users that this could break some things, so maybe don't include or enable it OOTB. 

Basically just gonna use this issue to request suggestions for getting GLFW working on Wayland properly in this Flatpak, as I don't have much experience in Java.

**Comment by orowith2os** [comment]
https://github.com/flathub/org.prismlauncher.PrismLauncher/pull/17
